src/molgenisUiForm.js

```diff
@@ -1,7 +1,7 @@
 import FormComponent from './components/FormComponent'
-import EntityToStateMapper from './util/EntityToStateMapper'
+import MolgenisFormGenerator from './util/MolgenisFormGenerator'
```

Comment:
why is this rename needed , is the current name includes the 'Entity' this seems clear to me